zipkin-core/src/main/scala/org/http4s/zipkin/core/interpreters/collector/Http.scala

```diff
@@ -0,0 +1,35 @@
+package org.http4s.zipkin.core.interpreters.collector
+
+import argonaut.Argonaut._
+import org.http4s.Uri.RegName
+import org.http4s.client.Client
+import org.http4s.zipkin.core.algebras.Collector
+import org.http4s.zipkin.core.interpreters._
+import org.http4s.zipkin.core.ZipkinInfo
+import org.http4s.{MediaType, Method, Request, Uri}
+
+import scalaz.concurrent.Task
+
+final case class Http(client: Client) extends Collector {
+
+  override def send(zipkinInfo: ZipkinInfo): Unit = {
+    val uri = Uri(
+      authority = Option(Uri.Authority(
+        host = RegName("localhost"),
+        port = Option(9411)
+      )),
+      path = "/api/v1/spans"
+    )
+    val payload = List(zipkinInfo).asJson.nospaces
+    val request = Request(Method.POST, uri)
+      .withContentType(Option(MediaType.`application/json`))
+      .withBody(payload)
+
+    client
+      // Fire and forget.
+      .fetch(request)(response => Task.now(()))
+      // We don't care about failure.
```

Comment:
oh wait.. this is the reporter side! sorry, the finagle link is pretty good, but so might be some details here https://github.com/openzipkin/zipkin-reporter-java/blob/master/core/src/main/java/zipkin/reporter/ReporterMetrics.java

FWIW we call this side reporter (takes spans ) and sender (throws them over the wall)